gc_tests/run_tests.rs

```diff
@@ -12,19 +12,42 @@ use std::{env, path::PathBuf, process::Command};
 use lang_tester::LangTester;
 use tempdir::TempDir;
 
+#[macro_use]
+extern crate lazy_static;
+
 static CRATE_NAME: &'static str = "gcmalloc";
 
-fn main() {
-    let sanitizers: Vec<String> = match env::var("SANITIZERS") {
-        Ok(ref s) => s.split(';').map(|s| s.to_string()).collect(),
-        Err(_) => Vec::new(),
+lazy_static! {
```

Comment:
Is there a good reason for these to be `lazy_static`? Couldn't they just be values returned by normal functions? `lazy_static` has a compile-time and run-time overhead, so if we don't need it, it would probably be good to avoid it.

Reply:
Good point. Fixed: 5aecf56216da33a541e92c20bbc5b0af69d322ec